### PR TITLE
Feature/init project 1

### PR DESCRIPTION
**Pull request by Logs00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 > Website of Anthem
 
+## 🏁 Install
+
+Just pull the repo and launch `index.html` in your favorite browser on desktop our on your mobile.
```
Comment (materazu) [inline README.md:13]: maybe a typo here?

Comment (Logs00) [reply README.md]: it's corrected ;)

Comment (materazu) [reply README.md]: Resolve the conversation then ;)

Comment (materazu) [reply README.md]: ⏬  button inside

**Pull request merged**